### DISCUSSION TITLE
Suggestion: Add option for debug log level in the Misc tab

### DISCUSSION THREAD

**Discussion by wechsler42** [borgbase/vorta#802] [Ideas]
**Is your feature request related to a problem? Please describe.**
I would appreciate an easy way to set the log level of vorta/borg to "debug"

**Describe the solution you'd like**
Add a checkbox to the Misc tab of vorta:
Set log level to debug (very detailed logging , needs more disk space)

**Comment by m3nu** [comment]
Most borg commands are already run with the `--debug`-option enabled and these debug-messages are written to Vorta's log.

What information are you missing specifically?

**Comment by wechsler42** [comment]
Thanks for the update,, so I did not know this.
Specifically I was missing some information on the repository check. So it was written that it started but nothing more. After a certain while I was wondering if it is stalled ot still working. When I do a repository check with borg on the CLI then I see there a note about how many percentages of the repository are already checked.

**Comment by m3nu** [comment]
There should be some information displayed. We call it with the `info` log-level:

`['borg', 'check', '--info', '--log-json']`

Should look like this:

<img width="587" alt="Screen Shot 2019-05-15 at 14 16 43" src="https://user-images.githubusercontent.com/3916435/57752543-254b6b00-771c-11e9-9e68-d6102aa47002.png">

**Comment by wechsler42** [comment]
Thanks m3nu for the feedback.
Well, it looks a bit different here (see the screenshot).
It stays with the message "INFO: Remote: Starting repository check" until the archive check is finished with "INFO: Archive consistency check complete, no problems found."
![vorta_archive_check](https://user-images.githubusercontent.com/19371424/57800779-7e35f600-7752-11e9-831d-8f9ea2d2c9fa.png)

**Comment by m3nu** [comment]
`borg check` has 2 parts: repo check and archive check. You are correct, that there is no log output for the first repo check part, only for the second archive check (lists archives).
